src/components/mgt-people-picker/mgt-people-picker.ts

```diff
@@ -454,13 +443,29 @@ export class MgtPeoplePicker extends MgtTemplatedComponent {
             <div class="people-person">
               ${this.renderTemplate('selected-person', { person }, `selected-${person.id}`) ||
                 this.renderSelectedPerson(person)}
-              <div class="CloseIcon" @click="${() => this.removePerson(person)}">\uE711</div>
+              <div class="CloseIcon" @click="${e => this.removePerson(person, e)}">\uE711</div>
             </div>
           `
       )}
     `;
   }
 
+  private handleFlyout() {
```

Comment:
It seems that this doesn't handle scenarios when the user has typed something

Reply:
7443ef6 fixed here